client/lib/upgrades/actions/purchases.js

```diff
@@ -9,66 +9,21 @@ import i18n from 'i18n-calypso';
  */
 import { action as ActionTypes } from '../constants';
 import Dispatcher from 'dispatcher';
-import olark from 'lib/olark';
-import purchasesAssembler from 'lib/purchases/assembler';
 import wp from 'lib/wp';
 
 const debug = debugFactory( 'calypso:upgrades:actions:purchases' ),
 	wpcom = wp.undocumented();
 
-const PURCHASES_FETCH_ERROR_MESSAGE = i18n.translate( 'There was an error retrieving purchases.' ),
-	PURCHASE_REMOVE_ERROR_MESSAGE = i18n.translate( 'There was an error removing the purchase.' );
-
 function cancelPurchase( purchaseId, onComplete ) {
```

Comment:
Do you mean create a Redux action for it? I agree that we should do this, but it probably makes sense to do it separately, as this PR just intends to remove the old Flux actions/store and hook up the new Redux actions/branch of state.